plugins/ssl/multi_ssl

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# -*- sh -*-
+
+: << =cut
+
+=head1 NAME
+
+multi_ssl - Plugin to monitor CERTificate expiration on multiple services and ports
+
+=head1 CONFIGURATION
+
+  [multi_ssl_*]
+    env.services www.service.tld blah.example.net_PORT
```

Comment:
Is there a specific reason why you picked the underscore as the separator between hostname and Port? A color would probably feel more familiar.